src/audio/volume.c

```diff
@@ -198,9 +198,6 @@ static struct comp_dev *volume_new(struct sof_ipc_comp *comp)
 		cd->tvolume[i] = VOL_ZERO_DB;
 	}
 
-	/* set volume min/max levels */
-	vol_set_min_max_levels(cd, ipc_vol->min_value, ipc_vol->max_value);
```

Comment:
please edit the commit message so that the subject identifies the change, e.g. "volume: fix issue with default setting being overwritten"
Also add some text to clarify why this is needed. There were a set of changes to the volume recently and I can't figure out if this is legit or not.

Reply:
I made another commit today - hope it is cleaner now.